foundation/docs mermaid plugin broken on windows

```
PS C:\Users\Johannes\test\0922> collie foundation docs --preview
Tip: To set a default foundation run
        $ collie config set-foundation <foundation>"
 ? Select a foundation » f
generating docs foundation "f" ...
parsing kit modules ...
parsing kit modules DONE 70ms
parsing compliance controls ...
parsing compliance controls DONE 10ms
generating kit module documentation ...
could not find compliance control cfmm/security-and-compliance/shared-responsibility-model-alignment referenced in a compliance statement in kit\foundation\docs\README.md
generating kit module documentation DONE 20ms
generate documentation foundations\f ...
parsing platform modules foundations\f\platforms\az ...
parsing platform modules foundations\f\platforms\az DONE 22ms
generate documentation foundations\f\platforms\az ...
generate documentation foundations\f\platforms\az DONE 12810ms
generate documentation foundations\f DONE 12848ms
generating docs foundation "f" DONE 13174ms

up to date, audited 341 packages in 13s

42 packages are looking for funding
  run `npm fund` for details

16 vulnerabilities (5 moderate, 11 high)

To address issues that do not require attention, run:
  npm audit fix

To address all issues possible (including breaking changes), run:
  npm audit fix --force

Some issues need review, and may require choosing
a different dependency.

Run `npm audit` for details.

> foundation-docs@1.0.0 docs:dev
> vuepress dev docs

warning plugin @vuepress/plugin-git has been used multiple times, only the last one will take effect
info Initializing VuePress and preparing data...
X [ERROR] [plugin vite:dep-scan] The argument 'path' must be a string, Uint8Array, or URL without null bytes. Received 'C:\\Users\\Johannes\\test\\0922\\foundations\\f\\.docs\\docs\\.vuepress\\.temp\\internal\\node_modules\\C:UsersJohannes\test\x0...

    node_modules/resolve/lib/sync.js:31:22:
      31 │         var stat = fs.statSync(dir, { throwIfNoEntry: false });
         ╵                       ^

    at Object.statSync (node:fs:1706:10)
    at isDirectory (C:\Users\Johannes\test\0922\foundations\f\.docs\node_modules\resolve\lib\sync.js:31:23)
    at loadNodeModulesSync (C:\Users\Johannes\test\0922\foundations\f\.docs\node_modules\resolve\lib\sync.js:200:17)
    at Function.resolveSync [as sync] (C:\Users\Johannes\test\0922\foundations\f\.docs\node_modules\resolve\lib\sync.js:107:17)   
    at resolveFrom (C:\Users\Johannes\test\0922\foundations\f\.docs\node_modules\vite\dist\node\chunks\dep-9c153816.js:2077:29)   
    at resolvePackageData (C:\Users\Johannes\test\0922\foundations\f\.docs\node_modules\vite\dist\node\chunks\dep-9c153816.js:28954:19)
    at C:\Users\Johannes\test\0922\foundations\f\.docs\node_modules\vite\dist\node\chunks\dep-9c153816.js:29335:15
    at Array.find (<anonymous>)
    at tryNodeResolve (C:\Users\Johannes\test\0922\foundations\f\.docs\node_modules\vite\dist\node\chunks\dep-9c153816.js:29334:44)
    at Context.resolveId (C:\Users\Johannes\test\0922\foundations\f\.docs\node_modules\vite\dist\node\chunks\dep-9c153816.js:29166:28)

  This error came from the "onResolve" callback registered here:

    node_modules/vite/dist/node/chunks/dep-9c153816.js:38656:18:
      38656 │             build.onResolve({
            ╵                   ~~~~~~~~~

    at setup (C:\Users\Johannes\test\0922\foundations\f\.docs\node_modules\vite\dist\node\chunks\dep-9c153816.js:38656:19)        
    at handlePlugins (C:\Users\Johannes\test\0922\foundations\f\.docs\node_modules\esbuild\lib\main.js:855:23)
    at Object.buildOrServe (C:\Users\Johannes\test\0922\foundations\f\.docs\node_modules\esbuild\lib\main.js:1149:7)
    at C:\Users\Johannes\test\0922\foundations\f\.docs\node_modules\esbuild\lib\main.js:2110:17
    at new Promise (<anonymous>)
    at Object.build (C:\Users\Johannes\test\0922\foundations\f\.docs\node_modules\esbuild\lib\main.js:2109:14)
    at Object.build (C:\Users\Johannes\test\0922\foundations\f\.docs\node_modules\esbuild\lib\main.js:1956:51)
    at C:\Users\Johannes\test\0922\foundations\f\.docs\node_modules\vite\dist\node\chunks\dep-9c153816.js:38432:54
    at Array.map (<anonymous>)

  The plugin "vite:dep-scan" was triggered by this import

    docs/.vuepress/.temp/internal/clientAppSetups.js:4:28:
      4 │ ...lientAppSetup3 from 'C:\Users\Johannes\test\0922\foundations\f\.docs\docs\.vuepress\theme\plugins\mermaid\client\...
        ╵                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~    

X [ERROR] [plugin vite:dep-scan] The argument 'path' must be a string, Uint8Array, or URL without null bytes. Received 'C:\\Users\\Johannes\\test\\0922\\foundations\\f\\.docs\\docs\\.vuepress\\.temp\\internal\\node_modules\\C:UsersJohannes\test\x0...

    node_modules/resolve/lib/sync.js:31:22:
      31 │         var stat = fs.statSync(dir, { throwIfNoEntry: false });
         ╵                       ^

    at Object.statSync (node:fs:1706:10)
    at isDirectory (C:\Users\Johannes\test\0922\foundations\f\.docs\node_modules\resolve\lib\sync.js:31:23)
    at loadNodeModulesSync (C:\Users\Johannes\test\0922\foundations\f\.docs\node_modules\resolve\lib\sync.js:200:17)
    at Function.resolveSync [as sync] (C:\Users\Johannes\test\0922\foundations\f\.docs\node_modules\resolve\lib\sync.js:107:17)   
    at resolveFrom (C:\Users\Johannes\test\0922\foundations\f\.docs\node_modules\vite\dist\node\chunks\dep-9c153816.js:2077:29)   
    at resolvePackageData (C:\Users\Johannes\test\0922\foundations\f\.docs\node_modules\vite\dist\node\chunks\dep-9c153816.js:28954:19)
    at C:\Users\Johannes\test\0922\foundations\f\.docs\node_modules\vite\dist\node\chunks\dep-9c153816.js:29335:15
    at Array.find (<anonymous>)
    at tryNodeResolve (C:\Users\Johannes\test\0922\foundations\f\.docs\node_modules\vite\dist\node\chunks\dep-9c153816.js:29334:44)
    at Context.resolveId (C:\Users\Johannes\test\0922\foundations\f\.docs\node_modules\vite\dist\node\chunks\dep-9c153816.js:29166:28)

  This error came from the "onResolve" callback registered here:

    node_modules/vite/dist/node/chunks/dep-9c153816.js:38656:18:
      38656 │             build.onResolve({
            ╵                   ~~~~~~~~~

    at setup (C:\Users\Johannes\test\0922\foundations\f\.docs\node_modules\vite\dist\node\chunks\dep-9c153816.js:38656:19)        
    at handlePlugins (C:\Users\Johannes\test\0922\foundations\f\.docs\node_modules\esbuild\lib\main.js:855:23)
    at Object.buildOrServe (C:\Users\Johannes\test\0922\foundations\f\.docs\node_modules\esbuild\lib\main.js:1149:7)
    at C:\Users\Johannes\test\0922\foundations\f\.docs\node_modules\esbuild\lib\main.js:2110:17
    at new Promise (<anonymous>)
    at Object.build (C:\Users\Johannes\test\0922\foundations\f\.docs\node_modules\esbuild\lib\main.js:2109:14)
    at Object.build (C:\Users\Johannes\test\0922\foundations\f\.docs\node_modules\esbuild\lib\main.js:1956:51)
    at C:\Users\Johannes\test\0922\foundations\f\.docs\node_modules\vite\dist\node\chunks\dep-9c153816.js:38432:54
    at Array.map (<anonymous>)

  The plugin "vite:dep-scan" was triggered by this import

    docs/.vuepress/.temp/internal/clientAppEnhances.js:5:30:
      5 │ ...entAppEnhance4 from 'C:\Users\Johannes\test\0922\foundations\f\.docs\docs\.vuepress\theme\plugins\mermaid\client\...
        ╵                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Error: Build failed with 2 errors:
node_modules/resolve/lib/sync.js:31:22: ERROR: [plugin: vite:dep-scan] The argument 'path' must be a string, Uint8Array, or URL without null bytes. Received 'C:\\Users\\Johannes\\test\\0922\\foundations\\f\\.docs\\docs\\.vuepress\\.temp\\internal\\node_modules\\C:UsersJohannes\test\x0...
node_modules/resolve/lib/sync.js:31:22: ERROR: [plugin: vite:dep-scan] The argument 'path' must be a string, Uint8Array, or URL without null bytes. Received 'C:\\Users\\Johannes\\test\\0922\\foundations\\f\\.docs\\docs\\.vuepress\\.temp\\internal\\node_modules\\C:UsersJohannes\test\x0...
    at failureErrorWithLog (C:\Users\Johannes\test\0922\foundations\f\.docs\node_modules\esbuild\lib\main.js:1624:15)
    at C:\Users\Johannes\test\0922\foundations\f\.docs\node_modules\esbuild\lib\main.js:1266:28
    at runOnEndCallbacks (C:\Users\Johannes\test\0922\foundations\f\.docs\node_modules\esbuild\lib\main.js:1046:63)
    at buildResponseToResult (C:\Users\Johannes\test\0922\foundations\f\.docs\node_modules\esbuild\lib\main.js:1264:7)
    at C:\Users\Johannes\test\0922\foundations\f\.docs\node_modules\esbuild\lib\main.js:1377:14
    at C:\Users\Johannes\test\0922\foundations\f\.docs\node_modules\esbuild\lib\main.js:678:9
    at handleIncomingPacket (C:\Users\Johannes\test\0922\foundations\f\.docs\node_modules\esbuild\lib\main.js:775:9)
    at Socket.readFromStdout (C:\Users\Johannes\test\0922\foundations\f\.docs\node_modules\esbuild\lib\main.js:644:7)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:343:12)
Unexpected error executing "(cd foundations\f\.docs && npm run docs:dev)"
Command exited with code 1
Tip: run collie with --verbose and --debug flags for more details
```

--- a/kit/foundation/docs/template/docs/.vuepress/config.ts
+++ b/kit/foundation/docs/template/docs/.vuepress/config.ts
@@ -6,7 +6,6 @@ import {
   ViteBundlerOptions,
 } from "vuepress-vite";
 import type { SidebarConfig, NavbarConfig } from "@vuepress/theme-default";
-import pluginMermaid from "./theme/plugins/mermaid";
 
 const navbar: NavbarConfig = [
   { text: "Foundation", link: "/" },
@@ -99,7 +98,6 @@ export default defineUserConfig<DefaultThemeOptions, ViteBundlerOptions>({
     },
   },
   plugins: [
-    pluginMermaid,
     [
       "@vuepress/plugin-git",
       {

--- a/kit/foundation/docs/template/package.json
+++ b/kit/foundation/docs/template/package.json
@@ -8,7 +8,6 @@
   },
   "devDependencies": {
     "vuepress": "2.0.0-beta.37",
-    "@vuepress/plugin-search": "2.0.0-beta.37",
-    "vuepress-plugin-mermaidjs": "https://github.com/laoergege/vuepress-plugin-mermaidjs.git#ef3b7fadf07c16289bd971eb8b20a388eab61469"
+    "@vuepress/plugin-search": "2.0.0-beta.37"
   }
 }

